libethcore/Transaction.cpp

```diff
@@ -31,6 +31,12 @@ using namespace std;
 using namespace dev;
 using namespace dev::eth;
 
+namespace
+{
+int const c_chainIdInNonceBits = 64;
+u256 const c_nonceLowMask = 0xffffffffffffffff;
```

Comment:
Sure it has, but it converts it to multiprecision internally anyway, so will we save anything?